src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceBase.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading;
 using System.Data;
 using System.Threading.Tasks;
-using Kusto.Language;
+using Microsoft.Kusto.ServiceLayer.DataSource.DataSourceIntellisense;
```

Comment:
"DataSource" seems redundant. Can we just rename the namespace to: Microsoft.Kusto.ServiceLayer.DataSource.Intellisense;